index.html

```diff
@@ -43,35 +43,38 @@ <h2>Let's start building your software&hellip;</h2>
         <img src="/img/build-icon.svg">
         <h1>Dev + ops</h1>
         <p>
-            - we are the full team you need, but also integrate with existing teams
-            - experts in application design
-            - experts in infrastructure
-            - experts in user requirement elicitation
-            - experts in storage solutions
-            - experts in ux
-            - we put it together making it easy for developers to deploy
-            - reduces cost of introducing new developers, any developer can follow code to production
+            Our team has experts in every area needed to build and run software in your organization.
+            We have experts in application design, infrastructure and cloud, user requirements elicitation, databases, and user experience design.
+        </p>
+        <p>
+            We bring all of these expertises with us, when we work with you.
+            By employing the principles of DevOps, each of our developers is empowered to build, deploy and maintain software for you, meaning that we deliver quickly and with great stability.
         </p>
     </div>
     <div class="point-of-differentiation">
         <img src="/img/explore-icon.svg">
         <h1>Technical counselling</h1>
         <p>
-            - xp from many startups
-            - building software from your needs first/concrete biz situation (green field v existing system)
-            - technical decisions: cloud, processes (workflow), security, future proofing, stability (testing)
-            - advise your cto or act your cto
+            We have put together a team with a lot of experience from a broad set of software businesses.
+            We know the things that commonly go wrong when running a development department and building software&mdash;and we use that knowledge to help you avoid these mistakes.
+        </p>
+        <p>
+            We always start with your concrete business situation, and help you take the best steps forwards, whether you have an existing piece of software to elaborate on, or are building a new one.
```

Comment:
I think it sound better with '... or you are building a new one'

Reply:
God feedback! Jeg endte med:

> We always start with your concrete business situation.
> We help you take the best steps forwards, whether that is to extend existing software, or to build something new.